TheFoodieNetwork/app/src/main/java/com/fbu/thefoodienetwork/fragments/ComposeFragment.java

```diff
@@ -0,0 +1,198 @@
+package com.fbu.thefoodienetwork.fragments;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.RatingBar;
+import android.widget.Spinner;
+import android.widget.Toast;
+
+import androidx.fragment.app.Fragment;
+
+import com.fbu.thefoodienetwork.R;
+import com.fbu.thefoodienetwork.databinding.FragmentComposeBinding;
+import com.fbu.thefoodienetwork.models.ParseRestaurant;
+import com.fbu.thefoodienetwork.models.ParseReview;
+import com.fbu.thefoodienetwork.models.Restaurant;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+import com.parse.SaveCallback;
+
+import org.parceler.Parcels;
+
+import java.util.List;
+
+public class ComposeFragment extends Fragment implements AdapterView.OnItemSelectedListener {
+    private static final String TAG = "ComposeFragment";
+    private static final String ARG_RESTAURANT = "selectedRestaurant";
+    private FragmentComposeBinding binding;
+    private Restaurant mRestaurant;
```

Comment:
So, I'm not opposed to you using Hungarian notation, however, if we are planning on doing so, then we should do so consistently.  (ie, you're using `mRestaurant` instead of `restaurant` for your variable name.  The "m" that you have prepended on the variable name means that it's a member [or instance] variable.  This is something we do on Android at FB, but it's kind of an outdated style.)